### PR TITLE
Fix to issue 726

### PR DESCRIPTION
**Pull request by lemikev**
The fix consists of prefixing data nodes within 'grouping' by the path of the 'uses'.
test20 is added to verify this behavior.

### CODE REVIEW

File: pyang/plugins/sid.py [hunk 1]
```diff
@@ -640,13 +645,14 @@ def get_path(self, statement):
                         break
                     parent = parent.parent
 
-                if parent.i_module is None or parent.i_module != statement.i_module:
-                    path = "/" + statement.i_module.arg + ":" + statement.arg + path
-                else:
+                if (parent.i_orig_module is not None and parent.i_orig_module == statement.i_orig_module) or (parent.i_orig_module is None and prefix_module is not None and prefix_module == statement.i_orig_module):
```
Comment (fredgan) [inline pyang/plugins/sid.py:648]: Please make it no longer than 80 characters per line

File: test/test_sid/test-5-expected-ietf-constrained-voucher@2019-08-01.sid [hunk 2]
```diff
@@ -1,76 +1,76 @@
 {
   "assignment-ranges": [
     {
-      "entry-point": 2500,
+      "entry-point": 2500,
```
Comment (fredgan) [inline test/test_sid/test-5-expected-ietf-constrained-voucher@2019-08-01.sid:4]: Why is there a space character in the end?

File: pyang/plugins/sid.py (hunk 1 above)
Comment (lemikev) [reply pyang/plugins/sid.py:648]: Fixed

File: test/test_sid/test-5-expected-ietf-constrained-voucher@2019-08-01.sid (hunk 2 above)
Comment (lemikev) [reply test/test_sid/test-5-expected-ietf-constrained-voucher@2019-08-01.sid:4]: The file is generated with json.dumps(), I don't have the control on spaces. It seems that the behavior of this Lib changed between the generation of the original files and now.

**Pull request merged**